app/src/main/java/com/capstone/event_finder/fragments/FeedFragment.java

```diff
@@ -85,20 +88,30 @@ public void getAPIEvents() {
         String eventSearchRegion = "en_US";
         Long eventSearchRadiusFromUserInMeters = 40000L;
         String numberOfEventsToRetrieve = "10";
-        String categories = "music, visual-arts, performing-arts, film, lectures-books, fashion, food-and-drink, festivals-fairs, charities, sports-active-life, nightlife, kids-family, other";
         Long upcomingEventsOnly = (System.currentTimeMillis() / 1000L);
         RetrofitClient.getInstance().getYelpAPI().getEvents(eventSearchRegion,
                 numberOfEventsToRetrieve,
                 upcomingEventsOnly,
                 eventSearchRadiusFromUserInMeters,
-                categories,
+                null,
                 ParseUser.getCurrentUser().getString("zip")).enqueue(new Callback<JsonObject>() {
             @Override
             public void onResponse(@NonNull Call<JsonObject> call, @NonNull Response<JsonObject> response) {
-                if (response.isSuccessful() && response.body() != null) {
+                if (response.isSuccessful() && response.body() != null && !String.valueOf(convertToList(response.body())).equals("[]")) {
```

Comment:
🛠  This is another instance where we want to validate against what the underlying JSON represents rather than the JSON string itself.

`[]` represents an empty array, so rather than do a string comparison, we should convert this to a list and check if it's empty.